src/robusta/core/playbooks/playbooks_event_handler_impl.py

```diff
@@ -189,12 +189,12 @@ def __run_playbook_actions(
                 not registered_action
             ):  # Might happen if manually trying to trigger incorrect action
                 msg = f"action {action.action_name} not found. Skipping for event {type(execution_event)}"
-                execution_event.response = self.__error_resp(msg)
+                execution_event.response = self.__error_resp(msg, 4600)
```

Comment:
makes more sense to me to throw an exception when an error happens

Reply:
same